app/displayer.py

```diff
@@ -0,0 +1,28 @@
+from abc import ABC, abstractmethod
+
+from app.book import Book
+
+
+class BookDisplay(ABC):
+    @abstractmethod
+    def print_display(self, book: Book) -> None:
+        pass
+
+
+class ConsoleBookDisplay(BookDisplay):
+    def print_display(self, book: Book) -> None:
+        print(book.content)
+
+
+class ReverseBookDisplay(BookDisplay):
+    def print_display(self, book: Book) -> None:
+        print(book.content[::-1])
+
+
+def create_book_displayer(display_type: str) -> BookDisplay:
+    if display_type == "console":
+        return ConsoleBookDisplay()
+    elif display_type == "reverse":
+        return ReverseBookDisplay()
+    else:
```

Comment:
No need to use else statement because all previous conditions return something ;)